client/src/components/ProductCard.tsx

```diff
@@ -133,7 +133,10 @@ function ProductCard ({ product, bookmark, log }: Props) {
   const clickLikeHandler: React.MouseEventHandler<HTMLDivElement> = () => {
     const workingLog = log!;
     workingLog.liked_it = !isLiked;
-    editLog(workingLog);
+    axios.patch(`/api/users/${auth?.user.uuid}/logs/${log!.id}/edit/`, 
+      workingLog,
+      { headers: headers }
+    )
```

Comment:
Thank you for catching this missing functionality! Could you add a `.then()` on this axios request to invoke:
```javascript
editLog(workingLog);
setIsLiked(!isLiked);
```

It may not be totally necessary right now, but it will help us in the future with things like cacheing if `editLog` is called to update LogContext after sending the request~